src/sage/schemes/overview.py

```diff
@@ -117,9 +117,9 @@
   ::
 
         sage: R.<x> = ZZ[]
-        sage: S.<t> = R.quo(x^2+5)
-        sage: P.<X,Y,Z> = ProjectiveSpace(2, S)
-        sage: P(S)
+        sage: S.<t> = R.quo(x^2 + 5)                                                    # optional - sage.rings.number_field
+        sage: P.<X,Y,Z> = ProjectiveSpace(2, S)                                         # optional - sage.rings.number_field
+        sage: P(S)                                                                      # optional - sage.rings.number_field
```

Comment:
`S` is not really a number field, but a quotient of a polynomial ring.
```
sage: R.<x> = ZZ[]
sage: S.<t> = R.quo(x^2+5)
sage: type(S)
<class 'sage.rings.polynomial.polynomial_quotient_ring.PolynomialQuotientRing_domain_with_category'>
```

Reply:
Indeed, fixed in 5065211aa9